package.json

```diff
@@ -50,35 +50,41 @@
   },
   "dependencies": {
     "@babel/polyfill": "^7.0.0",
```

Comment:
We might need also 
```json
"@babel/runtime-corejs2": "7.0.0",
```
as a dependency. Now, I think about it, it probably should be `dependency` inside Gutenberg as it is referenced inside generated JavaScript.